.github/workflows/create-collection.yml

```diff
@@ -8,6 +7,9 @@ on:
         description: 'OpenAPI filepath'     
         required: true
         default: 'adyen-openapi/yaml/CheckoutService-v70.yaml'
+      workspaceId:
+        description: 'Postman workspace ID'     
+        required: true
```

Comment:
Can you confirm this uses our hardcoded value by default, meaning we can just run the script and have both values set by default?